doc/changes/DM-45722.bugfix.md

```diff
@@ -0,0 +1 @@
+Explanatory logs for "initial data ID query returned no rows" now appear as a single entry in Loki/Json instead of being split on line breaks. There is no change to console behavior.
```

Comment:
I don't think Loki needs to be mentioned explicitly. I think you are saying "now appears as a single log message rather than multiple log entries".

Reply:
On second thought, I think I need at least an indirect reference to Loki, because the behavior there is what makes this a bugfix and not a coding style change. Is the new wording adequate?
> Explanatory logs for "initial data ID query returned no rows" now appear as a single log message instead of one entry per line. This improves display in log aggregators, but there is no change to console behavior.

Reply:
Works for me.